request-manager/graph/resolver.go

```diff
@@ -41,41 +40,44 @@ func NewResolverFactory(jf job.Factory, specs map[string]*spec.Sequence, seqGrap
 
 func (f *resolverFactory) Make(req proto.Request) Resolver {
 	return &resolver{
-		Request:           req,
-		JobFactory:        f.jf,
-		AllSequences:      f.seqs,
-		SequenceGraphs:    f.seqGraphs,
-		IdGen:             f.idf.Make(),
+		request:        req,
+		jobFactory:     f.jf,
+		allSequences:   f.seqs,
+		sequenceGraphs: f.seqGraphs,
```

Comment:
For end to end consistency (which also makes it easy to search for vars that are the same in various components): s/allSequences/seqSpecs/ and s/sequenceGraphs/seqGraphs/ and also above in `resolverFactory` s/seqs/seqSpecs/. Then we've got two consistent vars throughout: seqSpecs and seqGraphs.

Reply:
Similarly, we often use "seq" or "sequence" in var names. Let's use "seq" consistently because Go likes short var names. This is just for var names. In code comments, we can alternate between whichever reads best in context. I usually use full word first time it appears, then short form after.